frameworks/PHP/kumbiaphp/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+fw_depends mysql php7 nginx composer
+
+sed -i 's|localhost|'"${DBHOST}"'|g' bench/app/config/databases.php
```

Comment:
The raw connection use `TFB-database` and the ORM `DBHOST`, just for try both.
For speed, the best bet is an IP, since it avoid a name lookup. 
But it is insignificant, since all the frameworks use persistent connections.

It would be nice if all the frameworks used only one and avoid controversy. Add it to general requirements in https://www.techempower.com/benchmarks/#section=code

`DBHOST` -> 200 https://github.com/TechEmpower/FrameworkBenchmarks/search?utf8=%E2%9C%93&q=DBHOST&type=
`TFB-database` -> 96 https://github.com/TechEmpower/FrameworkBenchmarks/search?utf8=%E2%9C%93&q=TFB-database&type=

Reply:
`DBHOST` was used first before `TFB-database` was a thing, and now every time I am fixing some other issue in a framework and I see it using `DBHOST`, I change it to use `TFB-database` instead because the code is always simpler that way.  We're not interested in measuring the difference between the two in this project.  Use `TFB-database`.

Reply:
:+1: